src/components/molecules/index.js

```diff
@@ -1,10 +1,16 @@
 //> Components
 // Import all components to export them for easy access from parent components
-import Footer from "./Footer";
-import Navbar from "./Navbar";
 import CookieModal from "./CookieModal";
+import Navbar from "./Navbar";
+import Footer from "./Footer";
+import ReceivedUser from "./ReceivedUser";
 
-export { Footer, Navbar, CookieModal };
+export {
+  CookieModal,
+  Footer,
+  Navbar,
+  ReceivedUser,
```

Comment:
Export in the same order as imported.
```suggestion
  CookieModal,
  Navbar,
  Footer,
  ReceivedUser,
```